motr/client_internal.h

```diff
@@ -443,7 +443,12 @@ struct m0_client_layout_ops {
 /** miscallaneous constants */
 enum {
 	/*  4K, typical linux/intel page size */
+#ifdef CONFIG_X86_64
 	M0_DEFAULT_BUF_SHIFT        = 12,
+#else  //aarch64
+	M0_DEFAULT_BUF_SHIFT        = 16,
+
```

Comment:
This empty space is not needed here, drop it.

Reply:
Removed